library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -378,23 +364,16 @@ Http::FilterHeadersStatus PlatformBridgeFilter::encodeHeaders(Http::ResponseHead
     return Http::FilterHeadersStatus::Continue;
   }
 
+  // Delegate to base implementation for request and response path.
   return response_filter_base_.onHeaders(headers, end_stream);
 }
 
 Http::FilterDataStatus PlatformBridgeFilter::decodeData(Buffer::Instance& data, bool end_stream) {
   ENVOY_LOG(trace, "PlatformBridgeFilter({})::decodeData(length:{}, end_stream:{})", filter_name_,
             data.length(), end_stream);
 
-  // Delegate to shared implementation for request and response path.
-  Buffer::Instance* internal_buffer = nullptr;
-  if (request_filter_base_.iteration_state_ == IterationState::Stopped &&
-      decoder_callbacks_->decodingBuffer()) {
-    decoder_callbacks_->modifyDecodingBuffer([&internal_buffer](Buffer::Instance& mutable_buffer) {
-      internal_buffer = &mutable_buffer;
-    });
-  }
-
-  return request_filter_base_.onData(data, end_stream, internal_buffer);
+  // Delegate to base implementation for request and response path.
```

Comment:
This is only the request path looks like

Reply:
It's both - i.e., `onData` is a common method implemented in the `FilterBase` superclass of both `RequestFilterBase` and `ResponseFilterBase`. The comment is just to explain that we're delegating to a common execution path. If there's a clearer way to state it, I'm also open to suggestions.